### PR TITLE
Horizontal

### PR DESCRIPTION
**Pull request by LeticiaBriscoe**
Please check code and give me feed back. Thanks

### CODE REVIEW

File: Main/Main.ino [hunk 1]
```diff
@@ -46,6 +46,8 @@ void dmpDataReady() {
 
 int lightZapper = 9; // store pin 9 as our light controller
 int arduinoNumber = 1; // which arduino are we working with?
+int myPitch = 0; // variable needed to horizontal to verrtical function
+float lightStrength = 0; // variable needed to horizontal to verrtical function
```
Comment (Levon2014) [inline Main/Main.ino:51]: The code is very good and neat keep up the good work.

File: Main/gyro.ino [hunk 2]
```diff
@@ -153,6 +156,23 @@ void detectThrow() {
...
+  }
+  else if (myPitch >= 91 && myPitch <= 180) { //if the  pitch is between 3 and 6 o clock
+    lightStrength = (myPitch - ((myPitch - 90)* 2)* 2.83333); //assign lightStrength a value which decreases the fade
+  }
+  else if (myPitch >= 181 && myPitch <= 270) { //if the  pitch is between 6 and 9 o clock
+    lightStrength = ((myPitch - 180)* 2.83333); //assign lightStrength a value which increases the fade
+  }
+  else if (myPitch >= 271 && myPitch <= 360) { //if the  pitch is between 9 and 12 o clock
+    lightStrength = (myPitch - ((myPitch - 270)* 2)* 2.83333); //assign lightStrength a value which decreases the fade
+  }
```
Comment (francissmedley) [inline Main/gyro.ino:172]: nice neat if statements and else if statements

**Comment by Machinewarlock** [comment]
Great work, looks good to me.

File: Main/gyro.ino [hunk 3]
```diff
@@ -14,7 +14,7 @@
 // from the FIFO. Note this also requires gravity vector calculations.
 // Also note that yaw/pitch/roll angles suffer from gimbal lock (for
 // more info, see: http://en.wikipedia.org/wiki/Gimbal_lock)
-//#define OUTPUT_READABLE_YAWPITCHROLL
+#define OUTPUT_READABLE_YAWPITCHROLL
 
 // uncomment "OUTPUT_READABLE_REALACCEL" if you want to see acceleration
```
Comment (kimberleynyam) [inline Main/gyro.ino:19]: Your comments are impeccably precise, I could learn a thing or two.

**Comment by aliblackwell** [comment]
This is excellent! Well done @Kellyana and team 👍

**Pull request merged**